project-management-infrastructure/src/main/java/life/qbic/projectmanagement/infrastructure/sample/openbis/OpenbisConnector.java

```diff
@@ -152,17 +155,6 @@ private List<ch.ethz.sis.openbis.generic.asapi.v3.dto.project.Project> searchPro
     return searchResult.getObjects();
   }
 
-  private List<Sample> searchSamplesByCode(String code) {
-    SampleSearchCriteria criteria = new SampleSearchCriteria();
-    criteria.withCode().thatEquals(code);
-
-    SampleFetchOptions options = new SampleFetchOptions();
-    SearchResult<Sample> searchResult =
-        openBisClient.getV3().searchSamples(openBisClient.getSessionToken(), criteria, options);
-
-    return searchResult.getObjects();
-  }
-
```

Comment:
why was this deleted?

Reply:
because it does not work, but throws an exception with our mix of API versions